.github/workflows/ci.yml

```diff
@@ -15,8 +15,8 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: ['3.12']
-        toxenv: [py312, pylint]
+        python-version: ['3.11' ,'3.12']
```

Comment:
Why you added 3.11 ?

Reply:
@irtazaakram told me that we need to add "support" for 3.12. Also, the edx platform currently has `3.11`. That's why I have added 3.11.